FlowCal/transform.py

```diff
@@ -118,6 +128,15 @@ def to_rfi(data, channels=None, max_range=None, amplification_type=None):
         the first element is zero, the amplification type is linear. If
         None, take `amplification_type` from
         ``data.amplification_type(channel)``.
```

Comment:
Changed my mind and added the `$PnE` reference as indicated in 818a29410dd35b40a66aecbfd9c0f8682b93fe8d.